index.html

```diff
@@ -109,6 +121,7 @@
 					}
 				});
 				requestFlows.send();
+				window.localStorage.setItem('FDToken', token);
```

Comment:
This should be opt-in and monitor a `remember me locally` check box.

Reply:
@sqweelygig added